oat-sa/extension-tao-mediamanager
# [INV-229] Error occurs upon clicking 'Authoring' for passage in IE browser

Related to : https://oat-sa.atlassian.net/browse/INV-229
 
Allow to load plugins from bundles. include missing scripts to bundle
  
#### How to test
 
- prepare an instance of TAO
- prepare a shared stimulus
- try to open shared stimulus authoring in IE 11
- check that there is no errors in the console
